solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/Feature.java

```diff
@@ -365,11 +364,6 @@ public DocIdSetIterator iterator() {
         return in.iterator();
       }
 
-      @Override
```

Comment:
@cpoerschke  Let's add a comment saying that we intentionally don't delegate twoPhaseIterator because it doesn't work, and that we don't know why, with a reference to the JIRA issue?
Other than that, let's commit this to protect users as-is.